<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWindow</class>
 <widget class="QWidget" name="SetupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>430</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="rollButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll</string>
   </property>
  </widget>
  <widget class="QLabel" name="healthValue">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Health</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Stress</string>
   </property>
  </widget>
  <widget class="QLabel" name="stressValue">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Melee</string>
   </property>
  </widget>
  <widget class="QLabel" name="meleeValue">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Ranged</string>
   </property>
  </widget>
  <widget class="QLabel" name="rangedValue">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Mech</string>
   </property>
  </widget>
  <widget class="QLabel" name="mechValue">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Tech</string>
   </property>
  </widget>
  <widget class="QLabel" name="techValue">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>390</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>17</y>
     <width>681</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Welcome to the character creation screen. You can create a new character or load in a previously played character from the top menu. &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;After choosing a name and special skill, press the &amp;quot;Roll&amp;quot; button to get a random set of attributes. When you are ready to &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>308</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="horizontalSpacing">
     <number>12</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="characterName">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Special Skill:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="specialSkills">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string> Kung Fu (adds + 2% to Melee Combat points)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Precise Aim (adds + 2% to Ranged Combat points)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Metal Crafting (adds + 2% to Mechanical Intelligence)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Network Hacking (adds +2% to Technical Intelligence)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chemical Knowledge (adds +2% to Health)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tai Chi (adds -2% to Stress)</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>rollButton</zorder>
  <zorder>healthValue</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>stressValue</zorder>
  <zorder>label_7</zorder>
  <zorder>meleeValue</zorder>
  <zorder>label_9</zorder>
  <zorder>rangedValue</zorder>
  <zorder>label_11</zorder>
  <zorder>mechValue</zorder>
  <zorder>label_13</zorder>
  <zorder>techValue</zorder>
  <zorder>continueButton</zorder>
  <zorder>textEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
